extensions/interactions/GraphInput/directives/GraphInput.js

```diff
@@ -440,6 +461,8 @@ oppia.directive('graphViz', [
                 $scope.onTouchInitialVertex(index);
               } else {
                 if ($scope.state.addEdgeVertex === index) {
+                  $scope.state.hoveredVertex = null;
+                  $scope.helpText = EDGE_HELP_TEXT.INITIAL;
```

Comment:
Can you internationalize these strings? (There should be additions to en.json and qqq.json)

Reply:
Done